docs/source/development/CDFs/cdf_requirements.rst

```diff
@@ -0,0 +1,343 @@
+#####################
+CDF File Requirements
+#####################
+
+The following page describes the necessary contents inside of a CDF file generated for the IMAP mission.
+
+For a more general introduction to CDF files, see :ref:`CDF Introduction`.
+
+For sample code that will generate a file that matches the below guidelines, see :ref:`IMAP xarray_to_cdf Example`
+
+******************************
+Global Attributes
+******************************
+
+Global attributes are used to provide information about the data set as an entity. Together with variables and variable attributes, the global attributes make the data correctly and independently usable by someone not connected with the instrument team, and hence, a good archive product.
+
+Global attributes that have been identified for use with IMAP data products are listed below. Additional Global attributes can be defined but they must start with a letter and can otherwise contain letters, numbers and the underscore character (no other special characters allowed). Note that CDF attributes are case-sensitive and must exactly follow what is shown here.
+
+ISTP Compliant Global Attributes are listed here: `https://spdf.gsfc.nasa.gov/istp_guide/gattributes.html <https://spdf.gsfc.nasa.gov/istp_guide/gattributes.html>`_, and notes about how they are used on IMAP are below -
+
+Data_type
```

Comment:
Would this be easier to follow as a table?
Something like three rows: Attribute Name, Valid Options, Description

It is always a pain to make rst tables though, so this doesn't have to happen now and can be a follow-up if someone does want to do that later.